js/src/configuration-wizard.js

```diff
@@ -3,11 +3,8 @@ import React from "react";
 import ReactDOM from "react-dom";
 
 // Required to make Material UI work with touch screens.
-import { OnboardingWizard } from "yoast-components";
-import { MessageBox } from "yoast-components";
+import ConfigurationWizard, { MessageBox, MailchimpSignup, ConnectGoogleSearchConsole } from "@yoast/configuration-wizard";
```

Comment:
The imported components are different from what used to be imported.

| `YoastSEO` | `@yoast/configuration-wizard`
|:---|:---|
|[MailchimpSignup](https://github.com/Yoast/wordpress-seo/blob/trunk/js/src/components/MailchimpSignup.js) | [MailchimpSignup](https://github.com/Yoast/javascript/blob/develop/packages/configuration-wizard/tools/config/interfaces/MailchimpSignup.js) |
| [ConnectGoogleSearchConsole](https://github.com/Yoast/wordpress-seo/blob/trunk/js/src/components/ConnectGoogleSearchConsole.js) | [ConnectGoogleSearchConsole](https://github.com/Yoast/javascript/blob/develop/packages/configuration-wizard/tools/config/interfaces/ConnectGoogleSearchConsole.js) |

Reply:
Oh. Than I mistakenly replaced them with the ones used in the `configuration-wizard`. Therefore, the components don't need to be exported in the `configuration-wizard` any more.